mkdocs/docs/verify-release.md

```diff
@@ -105,15 +105,17 @@ make test
 To run the full integration tests:
 
 ```sh
-make test-s3
+make test-integration
```

Comment:
Was there a reason why it was S3? I know in the past, I had flakiness issues with the `test-integration` setup/

Reply:
looks like at the time when this was written, only `test-s3` was available
https://github.com/apache/iceberg-python/blob/3ecba3eac56ee286b38f509112d38a00c0fbfae3/Makefile#L34

Reply:
I think here we can use
```
make test-coverage
```

This option will run all unit tests, integration tests, and tests for s3fs, adlfs, gcsfs. It can also generate a test coverage report

This is also the command used by the CI: https://github.com/apache/iceberg-python/blob/e74ec4a89715f93852dfce4bb29f19463b1d54ef/.github/workflows/python-ci.yml#L53

Reply:
sounds good! i'll update it